EasyPost/Models/API/PostageLabel.cs

```diff
@@ -36,9 +33,7 @@ public class PostageLabel : Resource
         [JsonProperty("label_zpl_url")]
         public string label_zpl_url { get; set; }
         [JsonProperty("mode")]
-        public string mode { get; set; }
-        [JsonProperty("updated_at")]
-        public DateTime? updated_at { get; set; }
+        public new string mode { get; set; }
```

Comment:
Question: Typo? We don't want a `new` string right?

If so, please do a "find all" as there appears to be a few of these that we don't want to miss.